File: python/dsa/arrays/arrayADT.py
class myArray:
    def __init__(self, tsize, usize):
        self.size = tsize
        self.used = usize
        self.ptr = [0] * tsize

    def setVal(self):
        for i in range(self.used):
            self.ptr[i] = i*i
        
    def show(self):
        for i in range(self.used):
            print(f"{self.ptr[i]} ")
    
arr  =  myArray(10, 4)
arr.setVal()
arr.show()

File: java/dsa/arrays/arrayADT.java

class myArray {
    int tsize;
    int usize;
    int[] ptr;

    myArray(int x, int y){
        tsize = x;
        usize = y;
        ptr = new int[tsize];
    }

    void setVal(){
        for(int i=0; i<usize; i++){
            ptr[i] = i*i;
        }
    }

    void display(){
        for(int i=0; i<usize; i++){
            System.out.println(ptr[i]);
        }
    }
}

public class arrayADT {
    public static void main(String[] args){
        myArray arr= new myArray(10, 3);
        arr.setVal();
        arr.display();
    }
}